src/main/stackpack/stackpack.conf

```diff
@@ -0,0 +1,31 @@
+name = splunk-topology
+displayName = "Splunk Topology"
+categories = ["Logs"]
+releaseStatus = AVAILABLE
+logoUrl = "logo.png"
+isNew = false
+isMultiConfig = true
+# information that will go in the overview section
+overviewUrl = "overview.md"
+detailedOverviewUrl = "detailed.md"
+# information that will go in the configuration section
+configurationUrls {
+  NOT_INSTALLED: "default.md"
+  PROVISIONING: "provisioning.md"
+  WAITING_FOR_DATA: "waitingfordata.md"
+  INSTALLED: "installed.md"
+  DEPROVISIONING: "deprovisioning.md"
+  ERROR: "error.md"
+}
+faqs = []
+steps = [
+  {
+    name = "sts_instance_url"
+    display = "Splunk API URL"
+    value = {
+      type = "text"
+    }
+  }
+]
+provision = "SplunkTopologyProvision"
+dependencies: {}
```

Comment:
I think we need a dependency on common since we reference urn:stackpack:common

Reply:
done.